src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -449,7 +450,7 @@ private function addSessionSection(ArrayNodeDefinition $rootNode)
             ->children()
                 ->arrayNode('session')
                     ->info('session configuration')
-                    ->canBeEnabled()
+                    ->{!class_exists(FullStack::class) && interface_exists(CsrfTokenManagerInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
```

Comment:
I'm not sure how yet but I believe we could find a better way than relying on a `Symfony\Component\Security` class to enable the session automatically. Maybe a meta-package containing a `Symfony\Meta\RequiresSession` class? 🤷‍♂️

Reply:
I think we should not go in that direction. If enabling the session has no downside (with #25699), we should just enable them by default. But I wouldn't change the default in 3.4. Thus this PR, which is back in the game as is for me.

Reply:
Yes, you're right that it costs nothing. The `interface_exists` check is what is bothering me. Can we check on the `SessionInterface` interface directly instead of the `CsrfTokenManagerInterface`?